build-integrations/sbt-1.0/project/Integrations.scala

```diff
@@ -3,7 +3,7 @@ import sbt.{RootProject, uri}
 object Integrations {
 
   val SbtSbt = RootProject(
-    uri("git://github.com/scalacenter/sbt.git#c84db3a3969e269aeed0e9d9f2f384a0b029b82c"))
+    uri("git://github.com/scalacenter/sbt.git#fd66dfa1d40a01634f8ce65e299ce7a27cdb247c"))
```

Comment:
This was updated to avoid compiling sbt when we generate the bloop config files.